src/insns/cadd_32bit.adoc

```diff
@@ -48,7 +48,8 @@ For <<CADD>>, the address is incremented by the value in
 For <<CADDI>>, the address is incremented by the immediate value
 `imm`.
 
-include::malformed_clear_tag.adoc[]
+NOTE: This instruction sets `cd.tag=0` if `cs1` 's bounds are <<section_cap_malformed,malformed>>,
```

Comment:
Why is ADD singled out?

Reply:
because they don't expand the bounds, so for cheaper implementation we're only including instructions which do expand the bounds

Reply:
They still need to perform a representability check

Reply:
this is true, but don't expand the bounds. A functional unit which does CADD/CADDI doesn't need to do anything with the bounds, and the expansion is expensive just for this check.
If the expansion has already been done then the check is cheap.